src/components/scenes/Staking/StakeOptionsScene.tsx

```diff
@@ -149,7 +149,6 @@ const StakeOptionsSceneComponent = (props: Props) => {
             }
             keyExtractor={(stakePolicy: StakePolicy) => stakePolicy.stakePolicyId}
             scrollIndicatorInsets={SCROLL_INDICATOR_INSET_FIX}
-            ListEmptyComponent={<ActivityIndicator color={theme.primaryText} size="large" />}
```

Comment:
This was added in the previous commit, but now removed. Don't add it in the first place?